File: python/jdfzoj/catch.cpp
#include <stdio.h>
#include <stdlib.h>
using namespace std;
const int maxn=1010;
int main()
{
	char s[]="curl http://www.lydsy.com/JudgeOnline/problem.php?id=1000 -o 1000.html";
	for(int i=1;i<=5;i++)
	{
		s[54]=s[62]=i+'0';
		for(int j=0;j<=9;j++)
		{
			s[55]=s[63]=j+'0';
			for(int k=0;k<=9;k++)
			{
				s[56]=s[64]=k+'0';
				for(int l=0;l<=9;l++)
				{
					s[57]=s[65]=l+'0';
					system(s);
				}
			}
		}
	}
}

File: python/catch.py
from selenium import webdriver
import sys


reload(sys) 
sys.setdefaultencoding('utf-8') 

browser = webdriver.Chrome()
browser.get('http://www.lydsy.com/JudgeOnline/loginpage.php')
username = browser.find_element_by_xpath("//input[@name='user_id']")
username.clear()
username.send_keys("I_Love_CA")
password = browser.find_element_by_xpath("//input[@name='password']")
password.clear()
password.send_keys("2333333")
browser.find_element_by_xpath("//input[@name='submit']").click()


for i in range(1000, 5000):
    out = '%d' %i
    url = "http://www.lydsy.com/JudgeOnline/problem.php?id=" + out
    outt = out + ".html"
    output = open(outt, "w")
    browser.get(url)
    output.write(browser.page_source)
    output.close()
    print i

# http://creativecommons.org/licenses/by-nc-sa/4.0/
